src/main/java/org/fcrepo/transform/transformations/LDPathTransform.java

```diff
@@ -157,15 +157,4 @@ public boolean equals(final Object other) {
     public int hashCode() {
         return Objects.hashCode(query);
```

Comment:
And with the move of so-called "fcrepo-transform" from the core to the asynch fabric, it won't be.